tools/src/main/scala/com/nvidia/spark/rapids/tool/qualification/QualificationArgs.scala

```diff
@@ -97,8 +108,10 @@ Usage: java -cp rapids-4-spark-tools_2.12-<version>.jar:$SPARK_HOME/jars/*
   }
 
   validate(filterCriteria) {
-    case crit if (crit.endsWith("-newest") || crit.endsWith("-oldest")) => Right(Unit)
-    case _ => Left("Error, the filter criteria must end with either -newest or -oldest")
+    case crit if (crit.endsWith("-filesystem") || crit.endsWith("-oldest")
```

Comment:
we should verify the entire string newest-filesystem, oldest-per-app-name, etc, otherwise we need to make sure its correct later.

Reply:
done